src/test/java/org/zeromq/TestDisconnectInproc.java

```diff
@@ -0,0 +1,94 @@
+package zmq;
```

Comment:
I missed that small one. Is there any reason why the test was moved from package zmq? Right now, it got namespace problem, and with ZMQ.ZMQ_X socket types...